src/main/resources/jenkins/scm/api/form/traits/traits.js

```diff
@@ -36,7 +36,7 @@
         if (c) {
             var btn = YAHOO.widget.Button.getButton(e.id);
```

Comment:
Yahoo buttons do have an `on` method https://yui.github.io/yui2/docs/yui_2.2.0/docs/YAHOO.util.Element.html#on I do not see anything here that indicates this change is needed....

Reply:
Thanks for clarification. closing this ticket as its not needed.